### PR TITLE
Change cloud from psi-pipelines to openstack

### PR DESCRIPTION
**Pull request by divyansh42**

### CODE REVIEW

File: clusters/hosted-mgmt/hive/pools/pipelines/pipelines-ocp-4-14-0-amd64-openstack-clusterpool.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ spec:
     tp.openshift.io/owner: pipelines
   platform:
     openstack:
-      cloud: psi-pipelines
+      cloud: openstack
```
Comment (2uasimojo) [inline clusters/hosted-mgmt/hive/pools/pipelines/pipelines-ocp-4-14-0-amd64-openstack-clusterpool.yaml:34]: I’m guessing this one needs to be reverted. I think it’ It indicates the name of the openstack cloud you’re connecting to.

**Comment by divyansh42** [comment]
@2uasimojo I have made the requested changes. Please take a look.

**Pull request merged**